fastparquet/api.py

```diff
@@ -416,19 +416,23 @@ def to_pandas(self, columns=None, categories=None, filters=[],
             inferred from the metadata (if this was originally pandas data); if
             the metadata does not exist or index is False, index is simple
             sequential integers.
-        row_filter: bool
+        row_filter: bool or tuple of a list and a ndarray
             Whether filters are applied to whole row-groups (False, default)
             or row-wise (True, experimental). The latter requires two passes of
             any row group that may contain valid rows, but can be much more
             memory-efficient, especially if the filter columns are not required
             in the output.
+            If a tuple, the first element has to be the list of row groups to
+            retrieve, and the second element has to be a ndarray of booleans
```

Comment:
The length of the ndarray must match the sum of the sizes of the row-groups. Should we check this up front?

Reply:
Yes, ok, taken into account in my last commit. I also extended the test case to test the error message.
Please, let me know if the message is clear enough.
I am fully aware my own writing is not so good, I certainly over complicate things sometimes.